src/main.rs

```diff
@@ -144,26 +109,26 @@ fn real_main(options: Options, config: &Config) -> CliResult {
     let ebuild_path = PathBuf::from(format!("{}-{}.ebuild", package.name(), package.version()));
 
     // Open the file where we'll write the ebuild
-    let mut file = try!(OpenOptions::new()
-                            .write(true)
-                            .create(true)
-                            .truncate(true)
-                            .open(&ebuild_path)
-                            .chain_err(|| "failed to create ebuild"));
+    let mut file = OpenOptions::new()
+        .write(true)
+        .create(true)
+        .truncate(true)
+        .open(&ebuild_path)
+        .unwrap();
```

Comment:
I wouldn't want this to panic when you failed to open the file. Returning an error saying it couldn't create the file would be nicer.